app/views/admin/physical_tickets/_physical_ticket.html.haml

```diff
@@ -0,0 +1,16 @@
+%tr
+  %td= physical_ticket.id
+  %td= physical_ticket.ticket.title
+  %td= physical_ticket.user.email
+  %td= humanized_money_with_symbol physical_ticket.ticket_purchase.amount_paid
+  %td
+    .btn-group
+      = link_to 'Show',
+      conference_physical_ticket_path(@conference.short_title,
```

Comment:
you shouldn't use `@conference` here, that's why rubocop is complaining. You can pass the conference title as a parameter when calling the partial :wink:

Reply:
Ok thanks. :smile: